pyproject.toml

```diff
@@ -46,15 +57,25 @@ column_limit = 80
 
 [tool.yapfignore]
 ignore_patterns = [
-  "venv/*",
+    "venv/*",
 ]
 
 [tool.isort]
 multi_line_output = 0
 line_length = 80
 include_trailing_comma = false
 
+[tool.pyright]
+autoSearchPaths=false
+extraPaths=[] # Include paths from PYTHONPATH env var and .env definition
+useLibraryCodeForTypes=true
+stubPath="./pylanceStubs"
```

Comment:
This and `ignore="pylanceStubs"` was needed because we were using some external stubs in the settings I showed you before. I don't think removing these two will change any behavior of pyright.

Reply:
You mean removing those two will NOT change any behavior?

Reply:
Yeah I meant that it wouldn't change the behavior. Sorry for the unclear phrasing.

Reply:
I removed the unnecessary settings. PTAL.